blazor/diagram/migration-classic-to-native.md

```diff
@@ -0,0 +1,577 @@
+---
+layout: post
+title: Migration from SfDiagram(classic) to SfDiagramComponent | Syncfusion
+description: Learn here all about how to migrate the SfDiagram to SfDiagramComponent in Syncfusion Blazor Diagram component and more.
+platform: Blazor
+control: Diagram Component
+documentation: ug
+---
+
+# Migration from SfDiagram(classic) to SfDiagramComponent control
+
+This article describes the API migration process of SfDiagramComponent from SfDiagram(classic).
```

Comment:
This article describes the API migration process of the SfDiagramComponent from SfDiagram(classic).